app/views/candidate_interface/application_form/_review.html.erb

```diff
@@ -1,91 +1,109 @@
+<section class="govuk-!-margin-bottom-8">
+  <% if !CandidateInterface::EndOfCyclePolicy.can_add_course_choice?(@application_form)%>
+    <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2"><%= t('page_titles.course_choices') %></h2>
+    <p class="govuk-body">
+      You’ll be able to find courses in <%= days_until_find_reopens %> <%= 'day'.pluralize(days_until_find_reopens) %>
+      (<%= EndOfCycleTimetable.find_reopens.to_s(:govuk_date) %>).
+      You can keep making changes to the rest of your application until then.
+    </p>
   <% else %>
-    <%= render(CandidateInterface::SectionMissingBannerComponent.new(section: 'efl', section_path: candidate_interface_english_foreign_language_start_path, error: missing_error)) %>
+    <h2 class="govuk-heading-m govuk-!-font-size-27">Courses</h2>
+    <%= render(
+          CandidateInterface::CourseChoicesReviewComponent.new(
+            application_form: application_form,
+            editable: editable,
+            heading_level: 3,
+            show_incomplete: true,
+            missing_error: missing_error,
+            application_choice_error: application_choice_error,
+            )
+        ) %>
   <% end %>
-<% end %>
+</section>
+
+<section class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('page_titles.references') %></h2>
+  <% if !application_form.enough_references_have_been_provided? && editable %>
+    <%= render(
+          CandidateInterface::SectionMissingBannerComponent.new(
+            section: :references_provided,
+            section_path: candidate_interface_references_review_path,
+            text: t('review_application.references_provided.incomplete', minimum_references: ApplicationForm::MINIMUM_COMPLETE_REFERENCES),
+            link_text: t('review_application.references_provided.complete_section'),
+            error: missing_error
+          ),
+          ) %>
+  <% else %>
+    <%= render(CandidateInterface::ReferencesReviewComponent.new(references: application_form.application_references.includes(:application_form).feedback_provided, editable: false)) %>
+  <% end %>
+</section>
+
+<section class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('page_titles.personal_details') %></h2>
+
+  <h3 class="govuk-heading-m"><%= t('page_titles.personal_information') %></h3>
+  <%= render(CandidateInterface::PersonalDetailsReviewComponent.new(application_form: application_form, editable: editable, missing_error: missing_error)) %>
+
+  <h3 class="govuk-heading-m"><%= t('page_titles.contact_details') %></h3>
+  <%= render(CandidateInterface::ContactDetailsReviewComponent.new(application_form: application_form, editable: editable, missing_error: missing_error, submitting_application: true)) %>
+</section>
+
+<section class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-m govuk-!-font-size-27">Qualifications</h2>
```

Comment:
Qualification sections should be ordered thus (alphabetical except other section, which comes last):

* Degree
* English GCSE or equivalent
* English as a foreign language assessment
* Maths GCSE or equivalent
* Science GCSE or equivalent
* Academic and other relevant qualifications

Reply:
👍